full-report/action.yml

```diff
@@ -28,17 +32,20 @@ runs:
         filenames: ${{ inputs.dist_filenames }}
         token: ${{ inputs.token }}
     - name: Generate SBOM File
+      # not all packages have third party dependencies, and so not all packages integrate with silk.
+      if: ${{ inputs.silk_asset_group }}
       uses: mongodb-labs/drivers-github-tools/sbom@v2
       with:
         silk_asset_group: ${{ inputs.silk_asset_group }}
     - name: Generate Sarif File
       uses: mongodb-labs/drivers-github-tools/code-scanning-export@v2
       with:
-        ref: ${{ inputs.release_version }}
+        ref: ${{ inputs.sarif_report_target_ref }}
         output-file: ${{ env.S3_ASSETS }}/code-scanning-alerts.json
     - name: Generate Compliance Report
-      uses: mongodb-labs/drivers-github-tools/compliance-report@v2
+      uses: baileympearson/drivers-github-tools/compliance-report@adjust-shared-actions-for-node
```

Comment:
Will revert this once I have approvals, leaving this as a reminder to do so

Reply:
Good idea. I did take a crack at fixing #37 and allow using the `github.action_repository` context, but that doesn't work unfortunately :(